browser/static/js/main_zstack.js

```diff
@@ -1301,10 +1301,23 @@ function action(action, info, frame = current_frame) {
 
         // load new value of seg_array
         // array of arrays, contains annotation data for frame
-        seg_array = payload.imgs.seg_arr;
+        if (payload.imgs.hasOwnProperty('seg_arr')) {
+          seg_arr = payload.imgs.seg_arr;
+        }
+
+        if (payload.imgs.hasOwnProperty('segmented')) {
+          segLoaded = false;
```

Comment:
I think we can just set `segLoaded` to `false` in this block, and remove line 1300 as well as the `else` case for loading images. (Similar thing to loading raw--we only need to set `rawLoaded` to `false` if we're changing the image source, otherwise it remains `true`)